### PR TITLE
[INLONG-4169][Sort] Add comments before classes in sort

### LINKED ISSUE #4169: [Improve][Sort] Add Java docs in sort
### Description

[Improve] Add comments in sort  modules

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EMsnap**
### Title Name: [INLONG-4169][Sort] Add comments before classes in sort
Fixes #4169 

### Motivation

currently there are many classes in sort that don't have enough comment

### Modifications

Add comments before classes in sort

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/ExtractNode.java [hunk 1]
```diff
@@ -35,6 +35,9 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ *
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/ExtractNode.java]: This  should be not empty?

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/Node.java [hunk 2]
```diff
@@ -33,6 +33,9 @@
 import java.util.Map;
 import java.util.TreeMap;
 
+/**
+ * Base class for extract node and load node
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/Node.java:37]: It also contains transform node?

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HiveLoadNode.java [hunk 3]
```diff
@@ -37,6 +37,9 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * hive load node with FlinkCdc
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HiveLoadNode.java]: Does hive node belong to flink cdc? Is it more appropriate to belong to flink connector?

Comment (EMsnap) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HiveLoadNode.java]: sure it's a typo, thanks

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/Node.java (hunk 2 above)
Comment (EMsnap) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/Node.java:37]: modified

**Review by yunqingmoswu — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Review by GanfengTan — APPROVED** [review]
Good

**Pull request merged**